src/test/java/seedu/address/logic/commands/expenditure/ExpEditCommandTest.java

```diff
@@ -58,54 +59,56 @@ public void execute_someFieldsSpecifiedUnfilteredList_success() {
         Expenditure editedExpenditure = personInList.withInfo(VALID_INFO_BOB).withId(VALID_ID_BOB)
                 .withTags(VALID_TAG_HUSBAND).build();
 
-        EditPersonDescriptor descriptor = new EditPersonDescriptorBuilder().withInfo(VALID_INFO_BOB)
+        ExpEditCommand.EditExpenditureDescriptor descriptor = new EditPersonDescriptorBuilder().withInfo(VALID_INFO_BOB)
```

Comment:
This part is what I meant from before, some other parts in the file are doing this, `ExpEditCommand.EditExpenditureDescriptor`